modules/mosdepth/main.nf

```diff
@@ -28,7 +29,13 @@ process MOSDEPTH {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def interval = window_size ? "--by ${window_size}" : "--by ${bed}"
+    if (window_size) {
+        interval = "--by ${window_size}"
+    } else if ( bed ) {
+        interval = "--by ${bed}"
+    } else {
+        interval = ""
```

Comment:
I assume this means `mosdepth` will run without any intervals too? Can't remember now if that is possible. Maybe we should add a test for these different scenarios?

Reply:
Yes, it does. More tests are a good suggestion though, I have added couple more tests to test the following scenarios now. 

1. No intervals
2. Intervals from a bed file
3. Intervals from window